package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@microsoft/teams-js",
   "author": "Microsoft Teams",
-  "version": "1.11.0",
+  "version": "1.13.0-beta.0",
```

Comment:
The latest stable version of the SDK is 1.11.0 and the latest beta version is 1.12.0-beta.3 so the next beta version should be 1.12.0-beta.4. We will only start on a 1.13-beta.0 once we've officially released 1.12.0.